File: java/reverse.java
// Username - Akash2001, aim: To reverse the string, date: 23-10-2023

import java.io.*;
import java.util.Scanner;
 
class reverse {
    public static void main (String[] args) {
       
        String str= "Hacktoberfest", nstr="";
        char ch;
       
      System.out.print("Original word: ");
      System.out.println(str);
       
      for (int i=0; i<str.length(); i++)
      {
        ch= str.charAt(i);
        nstr= ch+nstr;
      }
      System.out.println("Reversed word: "+ nstr);
    }
}

File: c++/reverse.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string input;
    
    cout << "Enter a string: ";
    cin >> input;

    // Reverse the string
    string reversed = "";
    for (int i = input.length() - 1; i >= 0; i--) {
        reversed += input[i];
    }

    cout << "Reversed string: " << reversed << endl;

    return 0;
}
